actionloop/build.gradle

```diff
@@ -15,5 +15,19 @@
  * limitations under the License.
  */
 
-ext.dockerImageName = 'actionloop'
+ext.dockerImageName = 'actionloop-v1.0.2'
```

Comment:
Do we really want to embed a full semantic version into the name of the image?   It's going to look odd when the thing is called actionloop-v1.0.2:x.y.z  (image name + semvar image version for proper apache release of this component).

Reply:
Yes I DO want to embed some versionin on the image, but I plan to do this only when something important changes. ActionLoop is now a bit special since there are now many runtimes depending on it and I cannot change it lightly. Now it is built statically and can be used in cases when the ordinary actionloop cannot be used. If you had a build with Alpine you HAVE to depend on actionloop-v1.0.2 since the ordinary one cannot. But I do not plan to change the version all the time, only when breaking changes happens.

Reply:
will move discussion to dev list.

Reply:
thanks for changing to v2.  lgtm